drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -46,6 +46,9 @@ static void sdw_compute_slave_ports(struct sdw_master_runtime *m_rt,
 	port_bo = t_data->block_offset;
 
 	list_for_each_entry(s_rt, &m_rt->slave_rt_list, m_rt_node) {
+		if (m_rt->direction == SDW_DATA_DIR_TX)
+			port_bo = t_data->block_offset;
```

Comment:
This seems to be only valid when all slaves on the link belong to the same group. Can we also check slave group id and use the same prot_bo for the slaves with the same group?

Reply:
@bardliao the stream is handled by a Dai-link, so if we have 2 slaves on the same link, they will be part of the same stream, so the code above is safe by construction. If we have 2 slaves on the same link that are completely independent, they would not be part of the same stream.

Reply:
Thanks @plbossart for the explanation